tests/test_browser.py

```diff
@@ -4752,3 +4752,7 @@ def test_minimal_runtime_loader_shell(self):
       for modularize in [[], ['-s', 'MODULARIZE=1']]:
         print(str(args + wasm + modularize))
         self.btest('minimal_hello.c', '0', args=args + wasm + modularize)
+
+  @no_fastcomp('offset converter is not supported on fastcomp')
+  def test_offset_converter(self, *args):
+    self.btest(path_from_root('tests', 'browser', 'test_offset_converter.c'), '1', args=['-s', 'USE_OFFSET_CONVERTER', '-g4'])
```

Comment:
Does this test that offset converter works from inside a worker?

Reply:
No, it didn't, but the latest commit fixes this.